### PR TITLE
OCPQE-25632: Add disaster recovery jobs for hypershift-aks

### PR DESCRIPTION
**Pull request by fxierh**
Add our first (infra-oriented) DR job for Hypershift on AKS. 

https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/56690/rehearse-56690-periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-disaster-recovery-infra-guest-f14/1834643040489181184
(manually short-circuited)

### CODE REVIEW

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-disaster-recovery-infra-guest-f14

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-disaster-recovery-infra-guest-f14

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-disaster-recovery-infra-guest-f14

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-disaster-recovery-infra-guest-f14

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-disaster-recovery-infra-guest-f14

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-disaster-recovery-infra-guest-f14

**Comment by fxierh** [comment]
/cc @liangxia 

Could you please take a look ?

File: core-services/ci-chat-bot/workflows-config.yaml [hunk 1]
```diff
@@ -331,6 +331,16 @@ workflows:
         name: "4.16"
         namespace: ocp
         tag: upi-installer
+  cucushift-installer-rehearse-azure-aks-hypershift-disaster-recovery-infra-guest:
```
Comment (liangxia) [inline core-services/ci-chat-bot/workflows-config.yaml:334]: Missing `platform` ?

Comment (fxierh) [reply core-services/ci-chat-bot/workflows-config.yaml:334]: Updated. Thanks !

**Pull request merged**